### PR TITLE
Update guidance regarding the private key format and encoding

### PR DESCRIPTION
**Pull request by jsvd**
1. clarify that the pkcs8 key should be PEM encoded
2. update the `openssl pkcs8` command link that was broken
3. provide an example of how to convert from PKCS1 to PKCS8

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -313,8 +313,13 @@ Time in milliseconds for an incomplete ssl handshake to timeout
   * There is no default value for this setting.
 
 SSL key to use.
-NOTE: This key need to be in the PKCS8 format, you can convert it with https://www.openssl.org/docs/man1.1.0/apps/pkcs8.html[OpenSSL]
-for more information.
+NOTE: This key needs to be in the PKCS8 format and PEM encoded. This conversion can be done using the https://www.openssl.org/docs/man1.1.1/man1/openssl-pkcs8.html[openssl pkcs8] 
+command. For example, the command to convert a PEM encoded PKCS1 private key to a PEM encoded, non-encrypted PKCS8 key is:
```
Comment (karenzone) [inline docs/index.asciidoc:317]:
```suggestion
This key must be in the PKCS8 format and PEM encoded. 
You can use the https://www.openssl.org/docs/man1.1.1/man1/openssl-pkcs8.html[openssl pkcs8] command to complete the conversion.
For example, the command to convert a PEM encoded PKCS1 private key to a PEM encoded, non-encrypted PKCS8 key is:
```

File: docs/index.asciidoc [hunk 2]
```diff
@@ -313,8 +313,13 @@ Time in milliseconds for an incomplete ssl handshake to timeout
...
+
+["source","sh",subs="attributes"]
```
[the remark this answers is not in the document]
Comment (karenzone) [reply docs/index.asciidoc:319]: The way you have it isn't incorrect. I use the simplest version that works to avoid propaging complexity.

**Comment by karenzone** [comment]
Looks like this now: 

Screen Shot 2022-01-18 at 6 34 10 PM

**Review by karenzone — APPROVED** [review]
Nice improvements to both the content and formatting. Thank you!

**Comment by jsvd** [comment]
bump patch

**Pull request merged**